test/readers/iso19115_3/testData/iso19115-3.xml

```diff
@@ -257,6 +257,19 @@
                      </gex:northBoundLatitude>
                   </gex:EX_GeographicBoundingBox>
                </gex:geographicElement>
+               <gex:geographicElement>
+                  <gex:EX_BoundingPolygon>
+                     <gex:polygon>
+                        <gml:Polygon gml:id="A1">
+                           <gml:exterior>
+                              <gml:LinearRing>
+                                 <gml:posList> 1 1 1 2 2 2 0 4 1 1 </gml:posList>
+                              </gml:LinearRing>
+                           </gml:exterior>
+                        </gml:Polygon>
```

Comment:
Ok I know what a polygon is, but what is a linearRing?

Reply:
[gml:LinearRing](http://www.datypic.com/sc/niem21/e-gml32_LinearRing.html)? "describes the exterior edge of a quadrilateral" from another source.  thanks for bringing this up. I needed an example of `<gex:EX_BoundingPolygon>` and this is what I found so i just copy/pasted it ( i'm gonna work on that soon ). i double checked `gml` in the 19115-3 schema directory and it's [not there](https://github.com/ISO-TC211/XML/tree/master/schemas.isotc211.org/19115/-3) but chris sent me an example of the namespaces to include in the header of the doc and it includes gml (i.e .`xmlns:gml="http://www.opengis.net/gml/3.2"` ). this is an example of why this feature is kinda painstaking because there's a lot of research involved even on a simple module.